### ISSUE TITLE
[accepting ideas] money system + number of pokeballs

### ISSUE THREAD

**Issue by bukinoshita** [bukinoshita/pokemon-game#22]
trainer starts with 5000 dollars, which is okay to buy some pokeballs. But if you don't have any money or pokeball anymore, you can't play the game.

would be to have a way to earn money or items. I thought the idea of earning items/money weekly based on trainers rankings position. But still not sure if this is the best approach. It might suck have to have 1 week to start capture again. In the other side, I don't want to make the game too easy where you have 100000 pokeballs and start throwing non-stop.

**Comment by filipelinhares** [comment]
Maybe trainers can "sell" yours repeated pokemons to earn money.

**Comment by bukinoshita** [comment]
maybe, not sure yet tho... because I don't save repeated pokemons on the database. Not sure if worth to capture a pokemon (waste a pokeball) on a pokemon I already have. Maybe each user, earns N dollars every day(?)

**Comment by filipelinhares** [comment]
Yeah, you're right...

Maybe find money and pokeballs when catch a pokemons, as loots in RPG games(?).
Battles (#24) could be a good way to make money too.

> Earn N dollars every day. 
Keep consistency and make the game fair is the point here. I don't know if giving "free" money every day is a good alternative.
I like the ideia of giving money every week based on ranking position and some alternative way to "make money" during the week.

**Comment by bukinoshita** [comment]
Agreed! My first thought was to earn money when trying to catch a Pokémon or battling, just like the game. Each battle, no matter if win/lose it earns some money. But this only be available when `$ pokemon battle` is ready.

[more info — prize money](https://bulbapedia.bulbagarden.net/wiki/Prize_money)
